deploy/olm-catalog/keycloak-operator/8.0.2/keycloak-operator.8.0.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,349 @@
+                - routes
+              verbs:
+                - list
+                - get
+                - create
+                - update
+                - watch
+            - apiGroups:
+                - extensions
+              resources:
+                - ingresses
+              verbs:
+                - list
+                - get
+                - create
+                - update
+                - watch
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+                - prometheusrules
+                - podmonitors
+              verbs:
+                - list
+                - get
+                - create
+                - update
+                - watch
+            - apiGroups:
+                - integreatly.org
+              resources:
+                - grafanadashboards
+              verbs:
+                - get
+                - list
+                - create
+                - update
+                - watch
+            - apiGroups:
+                - apps
+              resourceNames:
+                - keycloak-operator
+              resources:
+                - deployments/finalizers
+              verbs:
+                - update
+            - apiGroups:
+                - policy
+              resources:
+                - poddisruptionbudgets
+              verbs:
+                - get
+                - list
+                - create
+                - update
+                - watch
+            - apiGroups:
+                - keycloak.org
+              resources:
+                - keycloaks
+                - keycloaks/status
+                - keycloaks/finalizers
+                - keycloakrealms
+                - keycloakrealms/status
+                - keycloakrealms/finalizers
+                - keycloakclients
+                - keycloakclients/status
+                - keycloakclients/finalizers
+                - keycloakbackups
+                - keycloakbackups/status
+                - keycloakbackups/finalizers
+                - keycloakusers
+                - keycloakusers/status
+                - keycloakusers/finalizers
+              verbs:
+                - get
+                - list
+                - update
+                - watch
+          serviceAccountName: keycloak-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+  keywords:
+    - Keycloak
+    - Identity
+    - Access
+  links:
+    - name: Documentation
+      url: https://github.com/keycloak/keycloak-operator/tree/8.0.2/docs
+    - name: Keycloak
+      url: https://www.keycloak.org/
+    - name: Keycloak Discourse
+      url: https://keycloak.discourse.group/
+  maintainers:
+    - email: keycloak-dev@lists.jboss.org
```

Comment:
@slaskawi is this mailing list still correct?

Reply:
That's a good catch!

I tried to update it to Google Groups in `8.0.1` release but it turns out they require an email address here (and there's no option to switch this to an arbitrary URL). I'll look if anything changed in `9.0.0` release.

For the time being, I would leave it as is - this is just a sync commit that pushes changes merged into Operatorhub.

Reply:
ok, perfect.